style.css

```diff
@@ -49,10 +52,11 @@ h1 span {
   color: #172b4d;
 }
 
-.aboutMe {
+.Headline {
   width: 375px;
   height: 650px;
-  background-image: url(/img/header-illsutration-mobile@2x.png);
+  background-image: url(/img/Background_header.png);
```

Comment:
- [Optional ] You have a small error in your image link, that's why the background image is not displayed correctly outside of your local machine. it should be `img/Background_header.png` instead of `/img/Background_header.png`.